vmdb/app/models/ems_refresh/save_inventory_orchestration_stacks.rb

```diff
@@ -15,11 +15,18 @@ def save_orchestration_templates_inventory(_ems, hashes, _target = nil)
       hashes.zip(templates).each { |hash, template| hash[:id] = template.id }
     end
 
+    def all_orchestration_stacks(ems)
+      if ems.respond_to?(:orchestration_stacks_with_nested)
+        ems.orchestration_stacks_with_nested(true)
+      else
+        ems.orchestration_stacks(true)
+      end
+    end
+
```

Comment:
@Ladas your spec test failed because of line 35 in this file. If you use `orchestration_stacks_with_nested` to get all stacks, in line 35 you should change to `stacks = save_inventory_multi(:orchestration_stacks_with_nested)`. 

I would suggest to let `orchestraton_stacks` to hold all nested stacks, and introduce `orchestration_stacks_without_nested` for root level stacks only.